Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -299,13 +299,12 @@ bool GLContextWidget::load_file(Core::File& file)
             texture_image = bitmap_or_error.release_value_but_fixme_should_propagate_errors();
     } else {
         auto response = FileSystemAccessClient::Client::the().try_request_file(window(), builder.string_view(), Core::OpenMode::ReadOnly);
-        if (response.is_error())
```

Comment:
Sweet Tuba! Any reason you nested this code instead of leaving it nice and flat?

Reply:
This is to make sure we still load the 3d model if the texture could not be loaded. Notice: I removed the early return.

Reply:
Sorry, I hadn't had coffee and misread the indentation level. Nice work!